content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md

```diff
@@ -0,0 +1,305 @@
+---
+layout: blog
+title: "Kubernetes 1.25: cgroup v2 升级到 GA"
+date: 2022-08-31
+slug: cgroupv2-ga-1-25
+---
+
+<!--
+layout: blog
+title: "Kubernetes 1.25: cgroup v2 graduates to GA"
+date: 2022-08-31
+slug: cgroupv2-ga-1-25
+-->
+
+<!--
+**Authors:**: David Porter (Google), Mrunal Patel (Red Hat)
+-->
+**作者**: David Porter (Google), Mrunal Patel (Red Hat)
+
+<!--
+Kubernetes 1.25 brings cgroup v2 to GA (general availability), letting the
+[kubelet](/docs/concepts/overview/components/#kubelet) use the latest container resource
+management capabilities.
+-->
+Kubernetes 1.25 将 cgroup v2 升级为一般可用性（GA），
+让 [kubelet](/zh-cn/docs/concepts/overview/components/#kubelet) 使用最新的容器资源管理能力。
+
+<!--
+## What are cgroups?
+-->
+## 什么是 cgroup？
+
+<!--
+Effective [resource management](/docs/concepts/configuration/manage-resources-containers/) is a
+critical aspect of Kubernetes. This involves managing the finite resources in
+your nodes, such as CPU, memory, and storage.
+
+*cgroups* are a Linux kernel capability that establish resource management
+functionality like limiting CPU usage or setting memory limits for running
+processes.
+-->
+有效的[资源管理](/zh-cn/docs/concepts/configuration/manage-resources-containers/)是 Kubernetes 的一个关键方面。
+这涉及管理节点中的有限资源，例如 CPU、内存和存储。
+
+*cgroups* 是一种 Linux 内核功能，可建立资源管理功能，
+例如限制 CPU 使用率或为正在运行的进程设置内存限制。
+
+<!--
+When you use the resource management capabilities in Kubernetes, such as configuring
+[requests and limits for Pods and containers](/docs/concepts/configuration/manage-resources-containers/#requests-and-limits),
+Kubernetes uses cgroups to enforce your resource requests and limits.
+
+The Linux kernel offers two versions of cgroups: cgroup v1 and cgroup v2.
+-->
+当你使用 Kubernetes 中的资源管理功能时，例如配置 
+[Pod 和容器的请求和限制](/zh-cn/docs/concepts/configuration/manage-resources-containers/#requests-and-limits)，
+Kubernetes 会使用 cgroups 来强制执行你的资源请求和限制。
+
+Linux 内核提供了两个版本的 cgroup：cgroup v1 和 cgroup v2。
+
+<!--
+## What is cgroup v2?
+-->
+## 什么是 cgroup v2？
+
+<!--
+cgroup v2 is the latest version of the Linux cgroup API. cgroup v2 provides a
+unified control system with enhanced resource management capabilities.
+
+cgroup v2 has been in development in the Linux Kernel since 2016 and in recent
+years has matured across the container ecosystem. With Kubernetes 1.25, cgroup
+v2 support has graduated to general availability.
+-->
+cgroup v2 是 Linux cgroup API 的最新版本,
+提供了一个具有增强的资源管理能力的统一控制系统。
+
+自 2016 年以来，cgroup v2 一直在 Linux 内核中进行开发，
+近年来在整个容器生态系统中已经成熟。 在 Kubernetes 1.25 中，
+对 cgroup v2 的支持已升级为一般可用。
```

Comment:
```suggestion
对 cgroup v2 的支持已升级为正式发布。
```
Refer to [glossary](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#%E6%9C%AF%E8%AF%AD%E5%AF%B9%E7%85%A7)